pages/collections/_type/_.vue

```diff
@@ -261,6 +277,24 @@
           values: [title],
           code: null
         };
+      },
+      relatedLinkGen(item) {
+        let id = '',
```

Comment:
This shorthand syntax is not one we use elsewhere. Replace with:
```js
let id;
let name;
```

Reply:
sure